Snowflake.Data.Tests/ConcatenatedStreamTest.cs

```diff
@@ -0,0 +1,189 @@
+﻿/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+namespace Snowflake.Data.Tests
+{
+    using NUnit.Framework;
+    using Snowflake.Data.Core;
+    using System;
+    using System.IO;
+    using System.Text;
+
+    [TestFixture]
+    class ConcatenatedStreamTest
+    {
+        Stream concatStream;
```

Comment:
maybe change the name of private member to _ concatStream

Reply:
Ok